app.rb

```diff
@@ -95,6 +69,8 @@ def create_student
     parent_permission = gets.chomp.downcase == 'y'
 
     @people << Student.new(@classroom, age, name, parent_permission: parent_permission)
+    puts
+    puts 'Student created successfully'
```

Comment:
- [ ] The essence of the Single Responsibility Rule is to ensure that functions/classes are responsible for a single job or thing. For example, in this application, the method for the user's input and input validation should be a different method from the method that creates the person or book. I would advise that you put this in a different method and call the method here. Kindly refer to this [documentation](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) 👍 💻 

#### Kindly implement this change for similar codes in the following methods:
- `create_book`
- `create_teacher`
- `create_rental`
- `list_rental`